### DISCUSSION TITLE
Is it ready to use in browser?

### DISCUSSION THREAD

**Discussion by bajrangCoder** [raphamorim/rio#466] [General]
I've been researching a suitable terminal frontend that works well on browsers, especially on touch devices. Currently, I'm using xterm.js, but it seems to be primarily mouse-centric, causing some features like selection to malfunction (and implementing them myself is challenging). However, I stumbled upon a promising project recently. After reviewing some pull requests, I'm confident that it supports touch events and boasts impressive performance. I've noticed that it also involves WebAssembly (WASM), indicating its compatibility with web browsers. Unfortunately, there are no available docs for this aspect.

Is it truly ready for use on browsers? If so, how can I integrate it? I only require the frontend components, as I already have a backend written in Node.js, although I'm in the process of rewriting it in Rust.

I'm keen to incorporate this into my project: [acodex](https://acode.app/plugin/bajrangcoder.acodex)

**Comment by d4h0** [comment]
@bajrangCoder: In case you didn't see it: https://github.com/raphamorim/rio/issues/25

Basically, Wasm doesn't seem to be usable yet.
